configure.ac

```diff
@@ -145,7 +145,7 @@ AC_ARG_ENABLE([profiling],
 AC_ARG_WITH([debug-verbose],
```

Comment:
minor: Since the `_FILE_OFFSET_BITS=64` value is defined, we could remove AC_SYS_LARGEFILE from the configure.ac file now